packages/shared/src/error.ts

```diff
@@ -67,6 +67,10 @@ export function isUserLockedError(err: any) {
   return isClerkAPIResponseError(err) && err.errors?.[0]?.code === 'user_locked';
 }
 
+export function isPasswordPwnedError(err: any) {
+  return isClerkAPIResponseError(err) && err.errors?.[0]?.code === 'form_password_pwned';
```

Comment:
How come we use the first error item ? Will this always be the case from FAPI ? I see that we are doing the same above, just asking tho.